### ISSUE TITLE
Create url and view for listing of all models of an app

### ISSUE THREAD

**Issue by JuniorBee** [oscarmlage/django-cruds-adminlte#36]
Everyone, including me expects to have a list of all models of an app inside its main url. For example if I have an app named testapp and it has models named test1, test2, test3 I have to be able to see all of these models under the 127.0.0.1:8000/testapp url.

**Comment by oscarmlage** [comment]
You're right @JuniorBee, we need to devel something like a dashboard or similar. Meanwhile what I'm doing now is just create a template `templates/adminlte/lib/_main_sidebar.html` and put there the needed html to surf between the models.

Take a quick look  to the demo app: https://github.com/oscarmlage/django-cruds-adminlte/blob/master/demo/demo/templates/adminlte/lib/_main_sidebar.html

Thanks for the idea, we'll take it in account in next release and - of course - remember that PRs are always welcomed ;) ;)

**Comment by JuniorBee** [comment]
@oscarmlage I want you to know that you are doing a great work. I strongly hope this package will be a standard among django developers. Everyone I know is in seek of user friendly admin for django. When you are planning to release this feature. Only three days remaining for my deadline. Maybe I can try to help you.

**Comment by oscarmlage** [comment]
@JuniorBee Thanks for your words. The project was born because of (imho) there is a lack of friendly UX/UI admin for Django and I think many people is looking forward something similar. On the other hand we are only 2 people (@luisza and me) dealing with this stuff so all is going slower than expected.

I won't be online 'til next Monday due some travels here so it definitely won't happens in next days unless I get some PR. I encourage you to try to contribute to the project, it would be so glad to get more active people over here.

**Comment by luisza** [comment]
@JuniorBee sorry it's not a django admin alternative, sorry there are a lot of effort and years of work need it for to be equal o similar in quality to django admin, but we do our most effort to have a good package.
